### DISCUSSION TITLE
[Projects Beta] Group by repository/assignee

### DISCUSSION THREAD

**Discussion by PythonCoderAS** [community/community#8833] [Projects and Issues] [answered]
Currently, I can only group by status. I would like to be able to group by the repository of issues/PRs or the users that are assigned to the issue/PR. I understand the latter can be difficult due to being able to assign to multiple people, but the former should be easy since it's only one possible value.

**Comment by SAGDREIS** [comment]
+351930601040 meu projeto foram protocolados pessoalmente okay obrigada Presidente.

**Reply by PythonCoderAS** [reply]
??? This is spam

**Comment by azenMatt** [comment, marked as answer]
Hi @PythonCoderAS - we just released both group by repository and assignee! 

You can see all the latest updates here - https://github.blog/changelog/2021-12-16-the-new-github-issues-12-16-update/

**Reply by PythonCoderAS** [reply]
Thank you!

**Reply by SAGDREIS** [reply]
Grupo por repositories? Fazem aplicativo ESTA que esta faltando.SONIA APARECIDA GONCALVES DOS REIS 648.712.746-91 (351)914162674

**Comment by SAGDREIS** [comment]
Hello JOSE APARECIDO DOS REIS Telephone 351 914162 674 Why does Wase not
include your correct home address?  Av Antonio Marques, nr° 9, ap 1, Canas
do Senhorim, Portugal, 3525-201, PT since October/2022 we have been living
here in Portugal 🇵🇹 at this address and since October 2021 we have
arrived in Portugal PT.  SONIA APARECIDA GONCALVES DOS REIS
648.712.746-91.  I request an exchange so that I can have more safety on
the road.  I've been waiting to return here in Portugal PT since October
2022. Another thing, we have a car, there's no need to advertise other
people's cars, let them take care of their car.  PERFECT?

SONIA APARECIDA GONCALVES DOS REIS 648.712.746-91



Em seg., 6 de nov. de 2023 07:34, SÔNIA APARECIDA GONÇALVES DOS REIS <
***@***.***> escreveu:

**Comment by SAGDREIS** [comment]
Hello JOSE APARECIDO DOS REIS Telephone 351 914162 674 Why does Wase not
include your correct home address?  Av Antonio Marques, nr° 9, ap 1, Canas
do Senhorim, Portugal, 3525-201, PT since October/2022 we have been living
here in Portugal 🇵🇹 at this address and since October 2021 we have
arrived in Portugal PT.  SONIA APARECIDA GONCALVES DOS REIS
648.712.746-91.  I request an exchange so that I can have more safety on
the road.  I've been waiting to return here in Portugal PT since October
2022. Another thing, we have a car, there's no need to advertise other
people's cars, let them take care of their car.  PERFECT?
SONIA APARECIDA GONCALVES DOS REIS 648.712.746-91



Em seg., 6 de nov. de 2023 20:36, SÔNIA APARECIDA GONÇALVES DOS REIS <
***@***.***> escreveu:

**Comment by SAGDREIS** [comment]
Hello JOSE APARECIDO DOS REIS Telephone 351 914162 674 Why does Wase not include your correct home address?  Av Antonio Marques, nr° 9, ap 1, Canas do Senhorim, Portugal, 3525-201, PT since October/2022 we have been living here in Portugal 🇵🇹 at this address and since October 2021 we have arrived in Portugal PT.  SONIA APARECIDA GONCALVES DOS REIS 648.712.746-91.  I request an exchange so that I can have more safety on the road.  I've been waiting to return here in Portugal PT since October 2022. Another thing, we have a car, there's no need to advertise other people's cars, let them take care of their car.  PERFECT?

**Comment by SAGDREIS** [comment]
Hello JOSE APARECIDO DOS REIS Telephone 351 914162 674 Why does Wase not include your correct home address?  Av Antonio Marques, nr° 9, ap 1, Canas do Senhorim, Portugal, 3525-201, PT since October/2022 we have been living here in Portugal 🇵🇹 at this address and since October 2021 we have arrived in Portugal PT.  SONIA APARECIDA GONCALVES DOS REIS 648.712.746-91.  I request an exchange so that I can have more safety on the road.  I've been waiting to return here in Portugal PT since October 2022. Another thing, we have a car, there's no need to advertise other people's cars, let them take care of their car.  PERFECT?
